h2o-r/demos/te_encoding_possible_bug_demo.R

```diff
@@ -0,0 +1,23 @@
+library(h2o)
+h2o.init()
+
+print("Category `a` in `ColA` is represented only by one fold.
```

Comment:
please add a note explaining the difference

Reply:
I've decided to remove it because we are about to expose Java's implementation to R and Python  and it will be fixed automatically